pkg/cmd/install.go

```diff
@@ -86,16 +91,61 @@ func MakeInstall() *cobra.Command {
 
 		fmt.Printf("Result: %s %s\n", string(res.StdOut), string(res.StdErr))
 
-		absPath, _ := filepath.Abs(localKubeconfig)
-		fmt.Printf("Saving file to: %s\n", absPath)
-
 		kubeconfig := strings.Replace(string(res.StdOut), "localhost", ip.String(), -1)
 
-		writeErr := ioutil.WriteFile(absPath, []byte(kubeconfig), 0600)
-		if writeErr != nil {
-			return writeErr
+		if merge {
+			// Create a temporary kubeconfig to store the config of the newly create k3s cluster
+			file, err := ioutil.TempFile("/tmp", "k3s-temp-*")
```

Comment:
Maybe you should use https://golang.org/pkg/os/#TempDir instead?

We can't guarantee `/tmp/` is always available, or desirable.